test/test-parse-function.cpp

```diff
@@ -233,6 +233,35 @@ TEST(test_parse, generator_function_statement) {
   }
 }
 
+TEST(test_parse, unused_arrow_expression) {
+  {
+    padded_string code(u8"async () => {}"_sv);
+    spy_visitor v;
+    parser p(&code, &v);
+    p.parse_and_visit_module(v);
+    EXPECT_THAT(v.visits, ElementsAre("visit_enter_function_scope",       //
+                                      "visit_enter_function_scope_body",  //
+                                      "visit_exit_function_scope",        //
+                                      "visit_end_of_module"));
+    EXPECT_THAT(v.errors, ElementsAre(ERROR_TYPE_OFFSETS(
+                              &code, error_unused_arrow_expression,  //
+                              where, strlen(u8""), u8"async")));
```

Comment:
Nit: You can write `0` instead of `strlen(u8"")`.
```suggestion
                              where, 0, u8"async")));
```